src/components/shared/card.tsx

```diff
@@ -53,13 +54,13 @@ const listStyles = (
 ): SerializedStyles => {
 	return css`
 		background: white;
-		margin-right: ${remSpace[3]};
-		flex: 0 0 15rem;
-		height: 100%;
+		margin-right: ${remSpace[2]};
+		flex: 0 0 42vw;
 		display: flex;
 		flex-direction: column;
 		justify-content: space-between;
 		border-top: ${borderColor(type, format)};
+		max-width: 160px;
```

Comment:
`10rem`?

Reply:
Would that not a problem with the font resizer? The cards could end up being massive?